scenes/game_elements/props/filling_barrel/filling_barrel.tscn

```diff
@@ -163,8 +163,10 @@ script = ExtResource("1_md0e3")
 [node name="AnimatedSprite2D" type="AnimatedSprite2D" parent="."]
 unique_name_in_owner = true
 position = Vector2(0, -21)
-sprite_frames = ExtResource("3_o1oxn")
-animation = &"filling"
+sprite_frames = ExtResource("2_vbewj")
+animation = &"shatter"
+frame = 10
+frame_progress = 1.0
```

Comment:
The changes in this scene were made by the editor, but aren't needed. The `filling_barrel.tscn` scene can continue showing the "filling" animation instead of the "shatter" animation. Actually, I think this is important for StoryQuests because all the existing ones aren't using the fragile vasels. Same for the first 3 rounds in the lore quest.

I think it is safe to restore this `filling_barrel.tscn` scene from latest main without affecting the PR.